common/src/truffle.rs

```diff
@@ -103,3 +104,32 @@ mod tests {
         }
     }
 }
+
+/// Deserialization implementation for compatibility with ABIs produced with
+/// solc v0.6. This is a known `ethabi` issue with an open PR for a fix:
+/// https://github.com/openethereum/ethabi/issues/185
+/// https://github.com/openethereum/ethabi/pull/187
```

Comment:
Should we wait for ethabi to update with the fix or is it important to update solc quickly?

Reply:
Good point, but that would mean that `ethcontract` doesn't support v0.6.0 compiled contracts. The PR has been open for a couple weeks without any discussion so I would prefer having this and just removing it once it gets merged.